src/components/Flows/FlowLogs.tsx

```diff
@@ -152,6 +156,40 @@ const LogsContainer = ({
     setAction(newAction);
   };
 
+  const fetchLogs = async () => {
+    setLogsLoaded(false);
+    (async () => {
+      try {
+        const data = await httpGet(
+          session,
+          `prefect/flow_runs/${flowRunId}/logs?task_run_id=${run.id}&offset=${Math.max(
+            flowRunOffset,
+            0
+          )}&limit=${flowRunLogsOffsetLimit + 1}`
+        );
```

Comment:
Somehow prefect throws error when `flowRunLogsOffsetLimit` is 201, it works when it is 200. Lets make sure we send to prefect a multiple of this `flowRunLogsOffsetLimit` (set to 200 ) & not +1. We can adjust the offset accordingly if needed.